js/views/mainView.js

```diff
@@ -25,6 +25,11 @@ define([
 		      //reinitialize the timer
 		      $("body").trigger("rightAnswer")
 		      console.log("Right Answer");
+			  
+			} else {
+			  $("body").trigger("wrongAnswer")
+		      console.log("Wrong Answer.");
```

Comment:
Get rid of console.logs in stuff that gets committed...  Generally this is just left over from debugging.